homeassistant/components/enocean/services.yaml

```diff
@@ -0,0 +1,24 @@
+teach_in_device:
+  name: Teach-In Device
+  description: Start the Teach-In-Process between the dongle and a device. During the teach-in normal packets are ignored
```

Comment:
What does the teach in process do, and why is this a service? Is it something that the user wants to automate?

Reply:
Hey Martin / Mr. Hjelmare,
the purpose of the process is to "pair" devices. Some devices need to be taught on which sender they have to listen to. As a user you do it once per device. The user pairs, for example a thermostat with the transceiver. After the pairing process was successful, the thermostat does not "throw" away telegrams from the taught-in sender.

You can do this with other software, too, of course.

The process is described in the technical specification: https://www.enocean-alliance.org/wp-content/uploads/2020/07/EnOcean-Equipment-Profiles-3-1.pdf, pages 17 and following

Reply:
Ok. Then it doesn't sound like something that should be a service. Services are built for automation purposes.

I'd rather think this should be a flow that the user navigates from the frontend.